pynicotine/gtkgui/ui/popovers/wishlist.ui

```diff
@@ -116,7 +117,7 @@
                                 <child>
                                   <object class="GtkLabel">
                                     <property name="visible">1</property>
-                                    <property name="label" translatable="yes">_Clear All...</property>
+                                    <property name="label" translatable="yes">Clear All...</property>
```

Comment:
- Removed: Alt+_C mnemonic from risky Clear All button because this is too similar to Copy.